frame/contracts/src/wasm/runtime.rs

```diff
@@ -291,13 +363,6 @@ fn read_sandbox_memory<E: Ext>(
 	ptr: u32,
 	len: u32,
 ) -> Result<Vec<u8>, sp_sandbox::HostError> {
-	charge_gas(
-		ctx.gas_meter,
-		ctx.schedule,
-		&mut ctx.trap_reason,
-		RuntimeToken::ReadMemory(len),
-	)?;
```

Comment:
doc needs to be updated too because gas is not charged anymore here (same for function below)

Reply:
You are right. I think I updated all relevant docs now.